canonical-web-and-design/jaas-dashboard
# Remove release banner

## Done

Remove release banner

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check code

Fixes #865
